File: weather-app/app/src/main/java/com/rperazzo/weatherapp/services/WeatherService.java
package com.rperazzo.weatherapp.services;

import com.rperazzo.weatherapp.model.FindResult;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface WeatherService {

    @GET("find")
    Call<FindResult> find(
            @Query("q") String cityName,
            @Query("units") String units,
            @Query("appid") String apiKey
    );

}

File: weather-app/app/src/main/java/com/rperazzo/weatherapp/MVVM/model/service/WeatherService.kt
package com.rperazzo.weatherapp.MVVM.model.service

import com.rperazzo.weatherapp.MVVM.model.entity.FindResult
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

public interface WeatherService {
    @GET("find")
    fun find(
            @Query("q") cityName: String,
            @Query("units") units: String,
            @Query("appid") apiKey: String
    ): Call<FindResult>;
}
